x-pack/plugins/observability_solution/infra/public/pages/metrics/hosts/hooks/use_metrics_charts.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import useAsync from 'react-use/lib/useAsync';
+import type { LensBreakdownConfig } from '@kbn/lens-embeddable-utils/config_builder';
+import { findInventoryModel } from '@kbn/metrics-data-access-plugin/common';
+
+export const useMetricsCharts = ({ dataViewId }: { dataViewId?: string }) => {
+  const model = findInventoryModel('host');
+
+  const { value: charts = [] } = useAsync(async () => {
+    const { cpu, disk, memory, network } = await model.metrics.getCharts();
+
+    return [
+      cpu.xy.cpuUsage,
+      cpu.xy.normalizedLoad1m,
+      memory.xy.memoryUsage,
+      memory.xy.memoryFree,
+      disk.xy.diskUsage,
+      disk.xy.diskSpaceAvailable,
+      disk.xy.diskIORead,
+      disk.xy.diskIOWrite,
+      disk.xy.diskReadThroughput,
+      disk.xy.diskWriteThroughput,
+      network.xy.rx,
+      network.xy.tx,
+    ].map((chart) => ({
+      ...chart,
+      layers: chart.layers.map((layer) =>
+        layer.type === 'series'
+          ? {
+              ...layer,
+              breakdown: {
+                type: 'topValues',
+                field: 'host.name',
+                size: 10,
```

Comment:
Just a side note: I am not sure if that's a valid concern but here we should use `20` as that's the max based on the table pagination: 
<img width="1911" alt="image" src="https://github.com/elastic/kibana/assets/14139027/595e5c20-7206-4505-8157-98b1ff2a7036">

Reply:
Maybe we should not hardcode it then? If we change the max number of items on the table we will probably forget about this.

Reply:
yup, it's not hardcoded anymore